components/org.wso2.carbon.identity.application.authenticator.biometric.servlet/src/main/java/org/wso2/carbon/identity/application/authenticator/biometric/servlet/servlet/BiometricServlet.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.application.authenticator.biometric.servlet.servlet;
+
+import com.google.gson.Gson;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.authenticator.biometric.servlet.BiometricServletConstants;
+import org.wso2.carbon.identity.application.authenticator.biometric.servlet.model.WaitStatus;
+import org.wso2.carbon.identity.application.authenticator.biometric.servlet.store.impl.BiometricDataStoreImpl;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Component class for implementing the Biometric servlet.
+ * GET calls(with query parameter:session data key) from the client web side and POST calls(with query parameters
+ * :session data key and signed challenge) from the client android app are handled here by
+ * updating the hashmap "updateStatus".
+ *
+ */
+public class BiometricServlet extends HttpServlet {
+
+    private static final long serialVersionUID = -2050679246736808648L;
+    private static final Log log = LogFactory.getLog(BiometricServlet.class);
+    private BiometricDataStoreImpl biometricDataStoreInstance = BiometricDataStoreImpl.getInstance();
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        if (!(request.getParameterMap().containsKey(BiometricServletConstants.INITIATOR))) {
+            if (log.isDebugEnabled()) {
+                log.debug("Invalid syntax as the query parameter for initiator is missing.");
+            }
+        } else { // If the initiator is not null, else block is executed.
```

Comment:
Let's get the comment under the `else` block(--> move it to the next new line).

Reply:
done